lib/geolocation/simulator.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var console = require('console')
+
+var Simulator = function () {
+  this.value = {
+    latitude: 0,
+    longitude: 0
+  };
+};
+
+Simulator.prototype.read = function (callback) {
+  this.value.latitude = Math.random() * 90;
```

Comment:
- 90 to be sometime in other hemisphere ;)

Reply:
Hmm, I'm not actually that familiar with the geographic coordinate system. Do you mean to say that this is wrong? Or that 90 should be a possible value? (I figured for a simulator it did not matter that it did not simulate _every_ possible value, so the random latitude value is 0 to non-inclusive 90, but I can make that inclusive if that is important.)

Reply:
sorry formating is bad  I wrote "- 90" for minus 90

it's a detail for now

Reply:
Ah, right. I wasn't clear how that would be represented, Wikipedia said something about it being between 0 and 90, and I think a suffix to indicate the hemisphere. I've updated it to allow for negative numbers as well now :)